cpp/ast.py

```diff
@@ -1428,6 +1428,7 @@ def _get_bases(self):
     def _get_class(self, class_type, templated_types):
         class_name = None
         class_token = self._get_next_token()
+        name_tokens = []
```

Comment:
Sure. I am running it from the repo here https://github.com/gtri/scrimmage.git

After cloning/cd'ing into the repo, this should recreate the issue:

```cppclean -I scrimmage/bindings/include -I include .```